compiler_gym/test_jotai.py

```diff
@@ -0,0 +1,36 @@
+import gym
```

Comment:
I don't think this feels needs to be commited

Reply:
Ok, I'll remove it.